open-banking-accelerator/components/consent-management/com.wso2.openbanking.accelerator.consent.mgt.dao/src/main/java/com/wso2/openbanking/accelerator/consent/mgt/dao/impl/ConsentCoreDAOImpl.java

```diff
@@ -445,6 +445,42 @@ public boolean updateConsentMappingStatus(Connection connection, ArrayList<Strin
         }
     }
 
+    @Override
+    public boolean updateConsentMappingPermission(Connection connection, Map<String, String> mappingIDPermissionMap)
+            throws OBConsentDataUpdationException {
+
+        int[] result;
+        String updateConsentMappingPermissionQuery =
+                sqlStatements.getUpdateConsentMappingPermissionPreparedStatement();
+
+        try (PreparedStatement updateConsentMappingPermissionPreparedStmt =
+                     connection.prepareStatement(updateConsentMappingPermissionQuery)) {
+
+            log.debug("Setting parameters to prepared statement to update consent mapping permissions");
+
+            for (String mappingID : mappingIDPermissionMap.keySet()) {
+                updateConsentMappingPermissionPreparedStmt.setString(1, mappingID);
```

Comment:
Here 1 should be permission, and 2nd is mapping id right?

Reply:
Fixed by https://github.com/wso2/financial-open-banking/pull/94/commits/6a5fdb018623c079a3633f252e427a440555d1a8